atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -128,7 +128,7 @@ public int execute(final AtlasDbServices services) {
             while (startRow.isPresent()) {
                 Stopwatch watch = Stopwatch.createStarted();
                 SweepResults results = sweepRunner.run(table, batchSize, cellBatchSize, startRow.get());
-                printer.info("Swept from {} to {} in table {} in {} ms, examined {} unique cells, deleted {} cells.",
+                printer.info("Swept from {} to {} in table {} in {} ms, examined {} unique cells, deleted {} stale values.",
```

Comment:
I got thrown off by the use of the word "cells" since cells is just a (key, columnname) pair, and not a (key, columnname, timestamp) tuple.  Could we do entries/